highfidelity/hifi
# Login Dialog UI improvements

- Fix for #[18366](https://highfidelity.manuscript.com/f/cases/18366/Bad-word-break-on-new-login-dialog-Show-Hide-password-glyph-moves-Login-spinner-misplaced)
## TEST PLAN

- Launch Interface
- If logged in, log out
- Click `File > Log In / Sign Up`
- Try a bad username/password combination
- [Dialog](https://user-images.githubusercontent.com/10539521/45848481-7321e780-bce3-11e8-9445-dd90288125c0.png) should look like so (with red text in between the sign in text and the username text field)
- This should look the same within HMD mode as well
- After toggling the `Show Password` eye in the password text field, the eye should not shift nor look visually distorted in any way
- Resizing the interface while login dialog is displayed should not change the contents of the login dialog
